cfgov/unprocessed/css/molecules/social-media.less

```diff
@@ -36,62 +36,88 @@
 */
 
 .m-social-media {
-  display: inline-block;
-
-  // This could be wrapped in the `m-social-media__share` modifier,
-  // but it's ok on its own too.
-  &_heading {
-    .heading-5();
-    display: inline-block;
-    // 0.25em subtracted to compensate for inline spacing
-    padding-right: unit(@grid_gutter-width / @font-size - 0.25em, em);
-    margin-bottom: 0;
-    vertical-align: middle;
+  //icon CSS needs to be separate to cover footer icons as well
+  .m-social-media_icon {
+    color: @gray-dark;
+    font-size: unit(@grid_gutter-width / @base-font-size-px, rem);
+    line-height: 1;
+    .u-link__colors( @gray-dark, @pacific-80 );
+    .u-link__no-border();
   }
 
-  .m-list {
+  &__share {
     display: inline-block;
-    vertical-align: middle;
 
-    .m-list_item {
+    .m-social-media_heading {
+      .heading-5();
+      display: inline-block;
       // 0.25em subtracted to compensate for inline spacing
-      margin-right: unit(
-        (@grid_gutter-width / 2) / @base-font-size-px - 0.25em,
-        em
-      );
-      margin-bottom: 0;
+      padding-right: unit(15px / @base-font-size-px - 0.25rem, rem);
+      margin: 0;
+      vertical-align: middle;
     }
 
-    .m-list_item:last-child {
-      margin-right: 0;
+    .m-list {
+      display: inline-block;
+      vertical-align: middle;
+
+      .m-list_item {
+        // 0.25em subtracted to compensate for inline spacing
```

Comment:
We shouldn't do math with the denominator, only the numerator (if absolutely needed). The reason to use this `unit(…)` mixin is twofold: (a) to make it easier to see the intended pixel spacing and (b) to proportionally grow/shrink things if the user changed their base font size in their browser settings.

So you'd want `15px` to be the intended pixel value you see in the browser. Subtracting from `@base-font-size-px - 0.25rem` no longer makes 15px the intended pixel value.

Given:
- `@base-font-size-px` = `16px`
- `@base-font-size-px` × `0.25rem` = `4px` (16 * 0.25)

It appears what you're intending here is `11px`, or `unit(11px / @base-font-size-px, rem);`

However, is this right? Our spacing is usually divisions of 5px. I would say go with `unit(10px / @base-font-size-px, rem);` for consistency, if @jenn-franklin is fine with it.

Reply:
Change implemented

Reply:
Doesn't look to have pushed?